internal/genkit/plugins/internal/openaiapi/convert.go

```diff
@@ -86,16 +89,13 @@ func convertMessages(messages []*ai.Message) ([]goopenai.ChatCompletionMessagePa
 	for _, m := range messages {
 		switch m.Role {
 		case ai.RoleSystem: // system
-			var parts []goopenai.ChatCompletionContentPartTextParam
+			var text string
 			for _, content := range m.Content {
-				if content.Text == "" {
-					continue
+				if content.Text != "" {
+					text += content.Text
 				}
```

Comment:
We are preferring early-return design pattern. How about using continue in this line?